File: 70-climbing-stairs/climbing-stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        if(n<=3):
            return n
        f=2
        s=3
        fib=0
        for i in range(4,n+1):
            fib=f+s
            f=s
            s=fib
        
        return fib
        

File: 70-climbing-stairs/climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        if(n<=3){
            return n;
        }
        int f=2,s=3;
        int res=0;
        for(int i=4;i<=n;i++){
            res=f+s;
            f=s;
            s=res;
        }
        return res;
    }
}